2025-01-24 16:24:32,655 INFO    MainThread:16972 [wandb_init.py:monkeypatch_ipython():576] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001D7343B2710>
2025-01-24 16:24:32,655 INFO    MainThread:16972 [wandb_init.py:init():743] calling init triggers
2025-01-24 16:24:32,655 INFO    MainThread:16972 [wandb_init.py:init():748] wandb.init called with sweep_config: {}
config: {'epochs': 20, 'batch_size': 64, 'learning_rate': 0.001}
2025-01-24 16:24:32,655 INFO    MainThread:16972 [wandb_init.py:init():776] starting backend
2025-01-24 16:24:32,655 INFO    MainThread:16972 [wandb_init.py:init():780] sending inform_init request
2025-01-24 16:24:32,659 INFO    MainThread:16972 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2025-01-24 16:24:32,660 INFO    MainThread:16972 [wandb_init.py:init():795] backend started and connected
2025-01-24 16:24:32,662 INFO    MainThread:16972 [wandb_run.py:_label_probe_notebook():1213] probe notebook
2025-01-24 16:24:32,666 INFO    MainThread:16972 [wandb_run.py:_label_probe_notebook():1223] Unable to probe notebook: 'charmap' codec can't decode byte 0x81 in position 1041593: character maps to <undefined>
2025-01-24 16:24:32,666 INFO    MainThread:16972 [wandb_init.py:init():888] updated telemetry
2025-01-24 16:24:32,666 INFO    MainThread:16972 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-01-24 16:24:33,816 INFO    MainThread:16972 [wandb_init.py:init():967] starting run threads in backend
2025-01-24 16:24:33,879 INFO    MainThread:16972 [wandb_run.py:_console_start():2409] atexit reg
2025-01-24 16:24:33,879 INFO    MainThread:16972 [wandb_run.py:_redirect():2259] redirect: wrap_raw
2025-01-24 16:24:33,879 INFO    MainThread:16972 [wandb_run.py:_redirect():2324] Wrapping output streams.
2025-01-24 16:24:33,879 INFO    MainThread:16972 [wandb_run.py:_redirect():2349] Redirects installed.
2025-01-24 16:24:33,880 INFO    MainThread:16972 [wandb_init.py:init():1009] run started, returning control to user process
2025-01-24 16:24:35,931 INFO    MainThread:16972 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-01-24 16:24:35,931 INFO    MainThread:16972 [wandb_init.py:_pause_backend():541] pausing backend
2025-01-24 16:27:38,408 INFO    MainThread:16972 [wandb_init.py:_resume_backend():546] resuming backend
2025-01-24 16:27:38,411 INFO    MainThread:16972 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-01-24 16:27:38,411 INFO    MainThread:16972 [wandb_init.py:_pause_backend():541] pausing backend
2025-01-24 16:27:39,241 INFO    MainThread:16972 [wandb_init.py:_resume_backend():546] resuming backend
2025-01-24 16:27:39,246 INFO    MainThread:16972 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-01-24 16:27:39,246 INFO    MainThread:16972 [wandb_init.py:_pause_backend():541] pausing backend
2025-01-24 16:27:45,213 INFO    MainThread:16972 [wandb_init.py:_resume_backend():546] resuming backend
2025-01-24 16:27:45,215 INFO    MainThread:16972 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-01-24 16:27:45,215 INFO    MainThread:16972 [wandb_init.py:_pause_backend():541] pausing backend
2025-01-24 16:27:53,159 INFO    MainThread:16972 [wandb_init.py:_resume_backend():546] resuming backend
2025-01-24 16:27:53,174 INFO    MainThread:16972 [wandb_init.py:init():743] calling init triggers
2025-01-24 16:27:53,175 INFO    MainThread:16972 [wandb_init.py:init():748] wandb.init called with sweep_config: {}
config: {'epochs': 20, 'batch_size': 64, 'learning_rate': 0.001}
2025-01-24 16:27:53,175 INFO    MainThread:16972 [wandb_init.py:init():768] wandb.init() called when a run is still active
2025-01-24 16:34:01,465 INFO    MainThread:16972 [wandb_run.py:_finish():2134] finishing run harshvardhan-singh-iiit-hyderabad/CannyNet-Train-eval/e50wz8ay
2025-01-24 16:34:01,466 ERROR   MainThread:16972 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-01-24 16:34:01,466 INFO    MainThread:16972 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-01-24 16:34:01,466 INFO    MainThread:16972 [wandb_init.py:_jupyter_teardown():558] cleaning up jupyter logic
2025-01-24 16:34:01,466 INFO    MainThread:16972 [wandb_run.py:_atexit_cleanup():2374] got exitcode: 0
2025-01-24 16:34:01,466 INFO    MainThread:16972 [wandb_run.py:_restore():2356] restore
2025-01-24 16:34:01,466 INFO    MainThread:16972 [wandb_run.py:_restore():2362] restore done
2025-01-24 16:34:04,653 INFO    MainThread:16972 [wandb_run.py:_footer_history_summary_info():3976] rendering history
2025-01-24 16:34:04,654 INFO    MainThread:16972 [wandb_run.py:_footer_history_summary_info():4008] rendering summary
2025-01-24 16:34:04,656 INFO    MainThread:16972 [wandb_run.py:_footer_sync_info():3937] logging synced files
